static/usr/lib/systemd/system/vdsm-network.service.in

```diff
@@ -8,6 +8,8 @@ After=libvirtd.service openvswitch.service NetworkManager.service
 Type=oneshot
 EnvironmentFile=-/etc/sysconfig/vdsm
 ExecStartPre=@BINDIR@/vdsm-tool dump-bonding-options
```

Comment:
`dump-bonding-option` needs to be first as it creates files that are necessary for setupNetworks.